.eslintrc.json

```diff
@@ -37,8 +37,8 @@
 	},
 	"rules": {
 		"@typescript-eslint/explicit-module-boundary-types": "off",
+		"@typescript-eslint/no-var-requires": "warn",
 		"no-console": "warn",
-		"no-unused-vars": "warn",
```

Comment:
Why did we drop this? for me I would move it to err before removing it lolol

Reply:
Good catch, not sure what happened there...